integrations/localhero/src/main/java/ch/stayhome/integrations/localhero/infrastructure/feign/LocalHeroChApi.java

```diff
@@ -1,12 +1,18 @@
 package ch.stayhome.integrations.localhero.infrastructure.feign;
 
+import ch.stayhome.integrations.localhero.model.LocalHeroCategory;
 import ch.stayhome.integrations.localhero.model.LocalHeroPost;
+import ch.stayhome.integrations.localhero.model.PagedWordPressResult;
 import feign.Param;
 import feign.RequestLine;
 
-import java.util.List;
 
 public interface LocalHeroChApi {
-    @RequestLine("GET /index.php?rest_route={restRoute}")
-    List<LocalHeroPost> findAll(@Param("restRoute") String restRoute);
+
+	@RequestLine("GET /wp-json/wp/v2/posts?page={page}&per_page={size}")
```

Comment:
@lugipfupf I was not able to get it working with the rest_route approach AND the pagination